app/build.gradle

```diff
@@ -28,28 +33,25 @@ gradle.projectsEvaluated {
     }
 }
 
-preBuild {
+preBuild.doLast {
+    def imlFile = file(project.name + ".iml")
 
-    doLast {
-        def imlFile = file(project.name + ".iml")
+    try {
+        def parsedXml = (new XmlParser()).parse(imlFile)
+        def jdkNode = parsedXml.component[1].orderEntry.find { it.'@type' == 'jdk' }
+        parsedXml.component[1].remove(jdkNode)
 
-        try {
-            def parsedXml = (new XmlParser()).parse(imlFile)
-            def jdkNode = parsedXml.component[1].orderEntry.find { it.'@type' == 'jdk' }
-            parsedXml.component[1].remove(jdkNode)
+        def sdkString = "Android API " + android.compileSdkVersion.substring("android-".length()) + " Platform"
+        new Node(parsedXml.component[1], 'orderEntry', ['type': 'jdk', 'jdkName': sdkString, 'jdkType': 'Android SDK'])
+        XmlUtil.serialize(parsedXml, new FileOutputStream(imlFile))
 
-            def sdkString = "Android API " + android.compileSdkVersion.substring("android-".length()) + " Platform"
-            new Node(parsedXml.component[1], 'orderEntry', ['type': 'jdk', 'jdkName': sdkString, 'jdkType': 'Android SDK'])
-            XmlUtil.serialize(parsedXml, new FileOutputStream(imlFile))
-
-        } catch (FileNotFoundException ex) {
-            ex.printStackTrace()
-        }
+    } catch (FileNotFoundException ex) {
+        ex.printStackTrace()
```

Comment:
This was just moved one indentation level to the left. The new Node() stuff doesn't seem to make a difference for me.